.ci/daily/Jenkinsfile

```diff
@@ -86,6 +92,51 @@ pipeline {
             ])
           }
         }
+        stage('Trigger Docker CE GHA') {
+          when {
+            // for testing - start
+            expression {
+              cambpmWithLabels('docker')
+            }
+            // for testing - end
+//            branch cambpmDefaultBranch();
+          }
+          agent {
+            kubernetes {
+              cloud "kubernetes"
+              label "${utils.envPrefix()}ci-infra-camunda-cloud-build_${env.JOB_BASE_NAME.take(15)}-${env.BUILD_ID}"
+              defaultContainer 'alpine'
+              yaml Podspec.getAgentPodspec(this, 'alpine-small', [
```

Comment:
not bad!

Reply:
@lwille do you think this code block can be extracted in a method in the `jenkins-global-shared-library`?